tests/PHPUnit/System/expected/test_reportLimiting__Goals.getItemsCategory_day.xml

```diff
@@ -2,19 +2,46 @@
 <result>
 	<row>
 		<label>Books Cat #0</label>
+		<nb_visits>5</nb_visits>
```

Comment:
The reason why we have more categories here as before is, that custom variables reports are limited by the report limiting in the test. The ecommerce reports do not have any limits when archived, so all categories are shown now. Not sure if that's something we should maybe change.